geopandas/tests/test_sindex.py

```diff
@@ -43,8 +57,8 @@ def test_point(self):
 
     def test_empty_point(self):
         s = GeoSeries([Point()])
-
-        assert s.sindex is None
+        with pytest.warns(FutureWarning):  # TODO: remove once deprecated
```

Comment:
Might be good to add a docstring here indicating that this tests that empty geometries result in an empty tree.